s3torchbenchmarking/src/s3torchbenchmarking/benchmark_utils.py

```diff
@@ -57,32 +51,64 @@ def summarize(self) -> dict:
             "max": round(np.percentile(window, 100), self.precision),
         }
 
-    def __repr__(self):
-        summary_str = json.dumps(self.summarize())
+    def __str__(self):
+        summary_str = json.dumps(self.summarize(), indent=2)
```

Comment:
Why indent rather than something like `sort_keys`? I can't imagine a multi-line output here to be very usable